packages/customize-widgets/src/components/inspector/block-inspector-button.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { MenuItem } from '@wordpress/components';
+import { speak } from '@wordpress/a11y';
+
+function BlockInspectorButton( {
+	isInspectorOpened = false,
+	onToggle = noop,
+	small = false,
+	...props
+} ) {
+	const speakMessage = () => {
+		if ( isInspectorOpened ) {
+			speak( __( 'Block settings closed' ) );
+		} else {
+			speak(
+				__(
```

Comment:
Ahh yeah that makes sense. I just copied most of the code from post editor without thinking much of it.

We should also hide the block editor when navigating to the inspector, but not unmount it. It's a little bit different from how the customizer's sections work now though, since mounting/re-mounting a block editor takes a significant amount of time.

Reply:
> I think that we should remove this speak() stuff and instead focus the first focusable element in the inspector when the inspector is opened, but cc. @tellthemachines or @talldan for opinions on whether there's a better pattern to follow.

Sounds like you might need to decide whether it's a popover. It's kinda sounding like a popover.

If it's a popover:
- the opening button will have `aria-expanded` and `aria-haspopup` specified correctly
- It'll focus when first opened
- Keyboard navigation would be constrained in the inspector
- Escape would close it.
- Focus loss would close it.

Reply:
This UI pattern already exists in the Customizer already so I think let's match that as best as we can.